bundles/org.openhab.voice.marytts/bnd.bnd

```diff
@@ -0,0 +1,15 @@
+Bundle-SymbolicName: ${project.artifactId}
+DynamicImport-Package: *
+Import-Package: org.osgi.framework;version="[1.8,2)"
+Require-Capability: 
+    osgi.extender:=
+      filter:="(osgi.extender=osgi.serviceloader.processor)",
+    osgi.serviceloader:=
+      filter:="(osgi.serviceloader=marytts.config.MaryConfig)";
+      cardinality:=multiple
+
+SPI-Provider: *
+SPI-Consumer: *
+
+-fixupmessages "Classes found in the wrong directory"; restrict:=error; is:=warning
+-fixupmessages "Unused Export-Package instructions"
```

Comment:
Are these fixups really used?

Reply:
Ok, I will try to remove whatever is not required. Line 15: yes, I explicitly added it to avoid the warning. Line 1 and 14 are copied over from jsscripting.

Reply:
Removing line 15 leads to warning
`bundles\org.openhab.voice.marytts\bnd.bnd [0:0]: Unused Export-Package instructions: [org.openhab.*]
`

Line 14 can be removed.